releasenotes/notes/add-scheduler-warnings-da701e21e53fbba6.yaml

```
@@ -0,0 +1,6 @@
+fixes:
+  - |
+    Give runtime warning for untrustworthy duration reports for mid-circuit measurements and resets.
+    No longer throw an error when Reset called in ConstrainedReschedule. Instead, we treat Reset like 
+    Measure.
```

Some small tweaks to the release note here:

```suggestion
    Fixed an issue where the :class:`.ConstrainedReschedule` transpiler pass would previously error
    if the circuit contained a :class:`~.circuit.Reset` instruction. This has been corrected so that the
    pass no longer errors, however as the durations reported by IBM backends are not always
    trustworthy for mid-circuit measurements and resets a ``RuntimeWarning`` is now raised if
    the :class:`.ConstrainedReschedule` is run on a circuit containing either.
    Fixed `#10354 <https://github.com/Qiskit/qiskit/issues/10354>`__
```